cli/ballerina-cli/src/main/resources/cli-help/ballerina-run.help

```diff
@@ -2,7 +2,7 @@ NAME
        ballerina-run - Compile and run the current package
 
 SYNOPSIS
-       bal run [OPTIONS] [<package>|<source-file>|<jar-file>] [--] <args...>
+       bal run [OPTIONS] [<package>|<source-file>|<jar-file>] [--] <(-Ckey=value)...>
```

Comment:
`args` and `configurables` are different and we support both. `args` are passed to the main function which is independent of the `configurables`. So the correct Synopsis would be `bal run [OPTIONS] [<package>|<source-file>|<jar-file>] [--] <args...> <(-Ckey=value)...>`

Reply:
Please update the rest of the help text accordingly